contributing.md

```diff
@@ -0,0 +1,163 @@
+# Contributing to pylabrad
+
+## Process
+
+The following instructions are for pylabrad contributors.
+Other users working in their own fork can will use whatever procedure they like for managing contributions.
+Once those contributions are ready to be included in the main pylabrad repository, they should file a pull request and allow the pylabrad contributors to carry out the rest of the steps.
+
+1. Most changes to the code start when a user makes an issue in the pylabrad issue tracker.
+This issue explains what feature is to be added or what bug is to be fixed.
+The issue is automatically assigned a unique number
```

Comment:
I would like to encourage people to just "show me the code", that is make a pull request and use that for discussion, rather than having a separate issue and pull request (in line with python's EAFP ethos). For cases where it's not clear how to solve the problem or we need to discuss before just starting to code, then beginning with an issue makes sense, but I think we should discuss both possibilities.